kalliope/core/CrontabManager.py

```diff
@@ -18,7 +18,7 @@ class InvalidCrontabPeriod(Exception):
     pass
 
 CRONTAB_COMMENT = "KALLIOPE"
-KALLIOPE_ENTRY_POINT_SCRIPT = "kalliope.py"
+KALLIOPE_ENTRY_POINT_SCRIPT = "__init__.py"
```

Comment:
did you test if the cronManager still working? We don't have tests for this yet.